src/components/Lesson/Lesson.js

```diff
@@ -15,6 +15,15 @@ const Lesson = React.createClass({
       __html: this.props.lesson.content
     };
   },
+  componentDidMount() {
+    // TODO: Remove scripts upon unmount?
+    if (containsScratchCode(this.props.lesson.content)) {
+      // TODO: use local scripts
+      appendScript('http://scratchblocks.github.io/js/scratchblocks-v3.1-min.js')
```

Comment:
The way it is now, these scripts are added every time you enter a lesson-page. Is it possible to
a) require them in Lesson.js instead?
If not, what about
b) including them in the index.html (i.e. adding the links in the `<head>` of index-template.ejs)?
If this does not work either, they should
c) be removed again in componentWillUnmount() (although I don't like this solution)

In addition I suggest we make an issue to create a better solution. I have searched a little, and found two interesting libraries:
1) https://github.com/mikenikles/html-to-react: Perhaps we can use this to convert the markdown html to react components, and perhaps avoid using dangerouslySetInnerHTML? This could open up several possibilities, including adding react components in the markdown (by adding appropriate functions in the processingInstructions array).
2) https://github.com/alexkuz/markdown-react-js: Instead of (or in addition to) converting markdown to html at compile time, we can do this "on the fly". In addition to allowing users to write markdown on the fly, it returns react components instead of html, and since it uses markdown-it and allows plugins, we should be able to create our own scratchblocks-plugin that would directly translate the scratchblocks-sections to scratchblocks-react-components (which in turn would be responsible for creating the scratchblocks-svg on-the-fly).

I especially hope the second idea is possible, and would like to pursue it.

But as I said, probably best to create a separate issue of this, and do it when the rest of the issues and PR's are resolved, just so that we get a working page to use this fall.

What do you think, @arve0 ?